server/pulp/server/db/manage.py

```diff
@@ -180,6 +185,12 @@ def ensure_database_indexes():
         model_class.ensure_indexes()
 
 
+def _get_running_worker_names():
```

Comment:
I think it might make more sense to put this in `status.get_running_workers()`
@bmbouter what do you think?